server/lib/template/helpers/dates.js

```diff
@@ -17,7 +17,7 @@ const TIMESTAMP_FMT = 'DD/MM/YYYY HH:mm:ss';
  * @returns {String} - the formatted string for insertion into templates
  */
 function date(value, dateFormat) {
-  dateFormat = dateFormat.name === 'date' ? DATE_FMT : dateFormat;
+  dateFormat = (!dateFormat || dateFormat.name === 'date') ? DATE_FMT : dateFormat;
```

Comment:
The `dateFormat = (dateFormat && dateFormat.name !== 'date') || DATE_FMT;` work well with the account module but broke the cashflow report.